packages/visx-demo/package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "build": "next build && next export",
     "deploy": "rm -rf out && yarn build && cd out && touch .nojekyll && git init && git add . && git commit -m \"Deploy commit\" && git remote add origin git@github.com:airbnb/visx.git && git push -f origin master:gh-pages",
-    "dev": "next",
+    "dev": "NODE_OPTIONS=--openssl-legacy-provider next",
```

Comment:
may need to update next, minimally need to add this same option to the build. don't fully understand why there is an error without this, because our version of next should be using webpack 5 by default 🤔

Reply:
I've tried to upgrade next to the latest version using `npx @next/codemod upgrade`  back when I made #1889 in an attempt to verify all components with react 19, but when I tried `babel-plugin-typescript-to-proptypes` did not work anymore (everything else did though), so I didn't continue working on it.

I can try again in the next few days based on this branch, maybe some other dependency upgrade made it work in the meantime, or maybe there is an easy replacement for that dependency...